sdk/tables/azure-data-tables/azure/data/tables/_policies.py

```diff
@@ -27,6 +27,12 @@
 def set_next_host_location(settings: Dict[str, Any], request: PipelineRequest) -> None:
     """
     A function which sets the next host location on the request, if applicable.
+
+    :param settings: The current retry context settings.
+    :type settings: dict[str, Any]
+    :param request: The outgoing request.
+    :type request: ~azure.core.pipeline.PipelineRequest
+    :return: None
```

Comment:
Do we need to add the rtype?

Reply:
I'm not sure as pylint not complaining without `rtype`. I think it makes sense not specifying the type for `None`.